dags/main_summary.py

```diff
@@ -203,15 +204,51 @@
     uri="https://raw.githubusercontent.com/mozilla/telemetry-airflow/master/jobs/retention.sh",
     dag=dag)
 
+client_count_daily_base = '''
```

Comment:
It might look nicer to represent this is a native list and concatenate using `"".join`.

Reply:
fixed